index.bs

```diff
@@ -5550,13 +5559,13 @@ Issue: Remove after HTML spec switched to [=WebDriver BiDi download will begin=]
    <code>timestamp</code> field set to |navigation info|["<code>timestamp</code>"], the
    <code>url</code> field set to |navigation info|["<code>url</code>"] and
    <code>suggestedFilename</code> field set to |navigation status|'s
```

Comment:
Is there actually a reason why you are using a mixture of `dict[name]` and `dict's name`? Shouldn't we always use the same approach? This applies to a lot of other places in this patch as well.